src/tests/cucumber/features/steps/common_steps/solver_output_handler.py

```diff
@@ -66,6 +66,10 @@ def __if_none_then_parse(self, rs: result_type, area, year, file_name: str):
     def __get_values_hourly(self, area: str, year: int):
         return self.__if_none_then_parse(result_type.VALUES, area.lower(), year, "values-hourly.txt")
 
+    def __get_values_hourly_for_specific_week(self, area: str, year: int, week: int):
+        df = self.__if_none_then_parse(result_type.VALUES, area.lower(), year, "values-hourly.txt")
+        return df[(df['hourly']['Unnamed: 1_level_1'] > (week - 1) * 168) & (df['hourly']['Unnamed: 1_level_1'] <= week * 168)]
+
```

Comment:
1. You can use file values-weekly.txt for that
2. I"m not sure I understand the special value `Unnamed: 1_level_1`

Reply:
ok but i think it should be done in another PR, to refactor these python scripts. the Unnamed level is because our output files contain empty column names